srcpkgs/yaz/template

```diff
@@ -13,7 +13,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://www.indexdata.com/resources/software/yaz/"
 distfiles="http://ftp.indexdata.dk/pub/yaz/yaz-${version}.tar.gz"
-checksum=311bcb386d6327dfa22e0e442aea26d1be7c633eb664b9f70d982f072a130112
+checksum=f0497fd8420574efab4e5738ea3b70787a6e8042f585156baa30bdc1911ba552
 
 case "$XBPS_TARGET_LIBC" in
 	musl) makedepends+=" libexecinfo-devel"
```

Comment:
The line below should be `LDFLAGS="-lexecinfo"`

Reply:
Sorry, which line are you referring to?